public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -194,12 +219,67 @@
 
     @Override
     public ResponseEntity<DbDomainListResponse> getDomainSearchResults(String keyword){
+
+        String[] keywords=keyword.split("[,+\\s+]");
```

Comment:
This looks like the same logic as in ConceptService. Refactor into a common place?